runtime/pyproject.toml

```diff
@@ -2,10 +2,11 @@
 requires = [
     "setuptools>=42",
     "wheel",
-    "cmake",
+    "cmake==3.25.0",
```

Comment:
Do we have to hard pin the cmake ? Sometimes the particular version is missing on various backends (macOS, linux, Windows, linux-aarch64).

Reply:
I've removed the pin. Was fiddling with it while trying to debug something that ended up not being load bearing.